specification/components/schemas/eRS-CommunicationRequest.yaml

```diff
@@ -0,0 +1,174 @@
+title: eRS-ReferralRequest
+type: object
+required:
+  - id
+  - meta
+  - resourceType
+  - contained
+  - extension
+  - status
+  - category
+  - subject
+  - recipient
+  - sender
+  - requester
+properties:
+  id:
+    type: string
+    example: '000000070000'
+  meta:
+    type: object
+    required:
+      - profile
+      - versionId
+    properties:
+      profile:
+        type: array
+        items:
+          type: string
+          enum:
+            - 'https://fhir.nhs.uk/STU3/StructureDefinition/eRS-CommunicationRequest-1'
+          example: 'https://fhir.nhs.uk/STU3/StructureDefinition/eRS-CommunicationRequest-1'
+      versionId:
+        type: string
+        example: '3'
+  resourceType:
+    type: string
+    enum:
+      - CommunicationRequest
+    example: 'CommunicationRequest'
+  contained:
+    type: array
+    nullable: true
+    description: |
+      | Field                                   | Cardinality | Notes                                      |
+      | --------------------------------------- | ----------- | ------------------------------------------ |
+      | `eRS-Shortlist-List`                    | `1..1`      |                                            |
+      | `eRS-ServiceSearchCriteria-Parameters`  | `1..1`      |                                            |
+      | `DocumentReference`                     | `0..*`      |                                            |
+      | `CareConnect-PractitionerRole`          | `1..1`      |                                            |
```

Comment:
I think you can have 1 or 2  PractitionerRole entries. One for the sender and one for the requester. Although they can reference the same one if it wasn't OBO.

Reply:
fixed